code/game/objects/items/storage/belt.dm

```diff
@@ -108,6 +108,15 @@
 	new /obj/item/extinguisher/mini(src)
 	new /obj/item/stack/cable_coil(src)
 
+/obj/item/storage/belt/utility/full/powertools/rcd/PopulateContents()
+	new /obj/item/screwdriver/power(src)
+	new /obj/item/crowbar/power(src)
+	new /obj/item/weldingtool/experimental(src)
+	new /obj/item/multitool(src)
+	new /obj/item/construction/rcd/loaded/upgraded(src)
+	new /obj/item/extinguisher/mini(src)
+	new /obj/item/stack/cable_coil(src)
+
```

Comment:
unmentioned change

Reply:
because it doesnt add or remove anything, its simply spawning the RCD in the belt instead of the backpack since the belt has a free slot lol

![image](https://user-images.githubusercontent.com/40489693/127753974-538da42e-9b61-4262-bf38-928c39f4458e.png)


on that note what did actually change was they have diagnostic sunglasses now instead of welding goggles

Reply:
that said, i think they dont have the RPD @Paxilmaniac so the engineer ERT wont have the tool needed to repair atmos problems lol

Reply:
the marine engi's backpack got removed, so i put their rcd in the belt, do i really need to mention something like that?

Reply:
no